src/main/java/study1/step2/Validator.java

```diff
@@ -0,0 +1,20 @@
+package study1.step2;
+
+public class Validator {
+    void validate(String input){
+        if(inputNullCheck(input) || inputValueCheck(input)){
+            throw new IllegalArgumentException("입력값 확인해주세요");
+        }
+    }
+
+    String[] getInputList(String input){
+        return input.split(" ");
+    }
+
+    private boolean inputNullCheck(String input){
+        return input == null;
+    }
+    private boolean inputValueCheck(String input) {
+        return "".equals(input.trim());
+    }
+}
```

Comment:
```suggestion
public class Validator {

    private final String input;
   
    private Validator (String input) {
        this.input = input;
    }
    
    public static Validator of(String input) {
        return new Validator(input);
    }
    
    public Validator validate(){
        if(inputNullCheck() || inputValueCheck()){
            throw new IllegalArgumentException("입력값 확인해주세요");
        }
        return this;
    }

    public String[] getTokens(){
        return input.split(" ");
    }

    private boolean inputNullCheck(){
        return input == null;
    }
    
    private boolean inputValueCheck() {
        return "".equals(input.trim());
    }
}
```

이렇게 만들 경우 다음과 같이 사용할 수 있습니다.

```java
        String[] inputList = Validator.of(input).validate().getTokens();
```